frontend/app/services/subscription-service.server.ts

```diff
@@ -92,7 +62,8 @@ function createSubscriptionService() {
       }),
     });
     instrumentationService.countHttpStatus('http.client.cdcp-api.users.gets', response.status);
-    const userParsed = userSchema.parse(await response.json());
+    const users = await response.json();
+    const userParsed = userSchema.parse(users._embedded.users[0]);
```

Comment:
This line may throw if there are no users returned.

Reply:
fixed